src/sentry/static/sentry/app/components/issueSyncListElement.jsx

```diff
@@ -0,0 +1,103 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+import InlineSvg from 'app/components/inlineSvg';
+import space from 'app/styles/space';
+
+class IssueSyncElement extends React.Component {
+  static propTypes = {
+    externalIssueLink: PropTypes.string,
+    externalIssueId: PropTypes.string,
+    onClose: PropTypes.func,
+    integrationType: PropTypes.oneOf(['github', 'jira', 'vsts']),
+  };
+
+  static defaultProps = {
+    extenalIssueLink: '#',
+  };
+
+  isLinked() {
+    return this.props.externalIssueLink && this.props.externalIssueId;
+  }
+
+  getIcon() {
+    switch (this.props.integrationType) {
+      case 'github':
+        return <IntegrationIcon src="icon-github" />;
+      case 'jira':
+        return <IntegrationIcon src="icon-jira" />;
+      case 'vsts':
+        return <IntegrationIcon src="icon-vsts" />;
+      default:
+        return null;
+    }
+  }
+
+  getText() {
+    return this.isLinked() ? (
+      <IntegrationLink href={this.props.externalIssueLink}>
+        {this.props.externalIssueId}
+      </IntegrationLink>
+    ) : (
+      <IntegrationLink href={'#'}>
+        Link <IntegrationName>{this.props.integrationType}</IntegrationName> Issue
+      </IntegrationLink>
+    );
+  }
+
+  render() {
+    return (
+      <IssueSyncListElementContainer>
+        <div>
+          {this.getIcon()}
+          {this.getText()}
+        </div>
+        <IconClose
+          src="icon-close"
+          onClick={this.props.onClose}
```

Comment:
Also it _might_ be worth using `remove` or `delete` instead of `close` because it could be confusing w/ the opening/closing of the modal for creating and linking issues.  

 `¯\_(ツ)_/¯`